postgres/src/test/scala/zio/sql/postgresql/SelectSpec.scala

```diff
@@ -0,0 +1,32 @@
+package zio.sql.postgresql
+
+import zio.Cause
+import zio.test.Assertion._
+import zio.test._
+
+object SelectSpec extends PostgresRunnableSpec with DbSchema {
```

Comment:
would you move this to PostgresSqlModuleSpec? as there are also tests for selects